assets/CET_to_py.py

```diff
@@ -77,6 +87,65 @@ def mpl_cm(name,colorlist):
     register_cmap("cet_"+name, cmap=cm[name])
     return cm[name]
 
+def get_aliases(name):
+    """Get the aliases for a given colormap name"""
+    for k, v in aliases.items():
+        if name == k or name == v:
+            name = '{0}, {1}'.format(v, k)
+    return name
+
+def all_original_names(group=None, not_group=None, only_aliased=False):
+    """Get all original names - optionally in a particular group - or only those with aliases"""
+    names = palette.keys()
+    if group:
+        names = filter(lambda x: group in x, names)
+    if not_group:
+        names = filter(lambda x: not_group not in x, names)
+    if only_aliased:
+        names = filter(lambda x: x in aliases.keys(), names)
+    else:
+        names = filter(lambda x: x not in aliases.values(), names)
+    return sorted(list(names))
+
+def colormap(name, cmap=None, bounds=None, array=None,**kwargs):
```

Comment:
I think `colormap` and `colormaps` were good names when these functions were in notebooks, because there they display a colormap or a collection thereof, but for the colorcet namespace those don't seem like the right names, given that colorcet is about colormaps and these functions don't return colormaps.  Maybe `cmap_plot` and `cmaps_plot`?  They aren't great names; maybe you can think of better.

Reply:
Yes I have been coming to that realization as well. Maybe just `plot` and `plots`?

Reply:
Fine by me, though maybe there is a name for this type of plot that would distinguish it from the candy buttons plot.

Reply:
Maybe swatch and swatches then like https://medium.com/@plotlygraphs/introducing-plotly-express-808df010143d